enterprise/api.py

```diff
@@ -7,10 +7,21 @@
 from enterprise.models import EnterpriseCustomerBrandingConfiguration
 
 
-def get_enterprise_branding_info(provider_id=None):
+def get_enterprise_branding_info(**kwargs):
```

Comment:
Do we really need to open this signature up to such a broad set of use cases?  It seems like below we are only concerned with two parameters -- we should be explicit about the arguments.